src/views/CreatePropertyView.vue

```diff
@@ -0,0 +1,80 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import ProjectService from '@/services/ProjectService';
+import InputText from 'primevue/inputtext';
+import Textarea from 'primevue/textarea';
+import Button from 'primevue/button';
+
+const props = defineProps<{
+  projectId: string;
+}>();
+
+const title = ref('');
+const description = ref('');
+const router = useRouter();
+const projectService = new ProjectService();
+
+const createProperty = () => {
+  console.log('createProperty called');
+  if (!title.value) {
+    console.log('Title is empty, no navigation occurs');
+    alert('Titel ist ein Pflichtfeld!');
+    return;
+  }
+
+  console.log('Calling projectService.createProperty');
+  projectService
+      .createProperty(props.projectId, {
+        title: title.value,
+        description: description.value,
+        landRegisterEntry: '',
+        plotArea: 0,
+        effective_space: 0,
+      })
+      .then(() => {
```

Comment:
async/await function would be a nicer alternative here, in my opinion. This is due to the fact that the rest of component uses modern TypeScript syntax and using async/ await would keep the consistency and better readibility of the code. However, .then() also works just fine, of course :)